basic-examples/quoting/README.md

```diff
@@ -1,15 +1,16 @@
-# Expressions
+# Quoting
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates example expressions with quotes as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+There are multiple types of quoting, and quotes must pass through YAML's syntax. Look at this example to learn about the various ways to determine what would work best for your use case.
 
 ## Files
 
 - [`workflow.yaml`](workflow.yaml) -- Defines the workflow input schema, the plugins to run
-  and their data relationships, and the output to present to the user
+  and their data relationships, and the output to present to the user.
 - [`input.yaml`](input.yaml) -- The input parameters that the user provides for running
-  the workflow
+  the workflow. Empty in this example.

```

Comment:
We have the space, so we might as well be a bit effusive, e.g.:
> The workflow input file is empty in this example, because the input for the subworkflow is constructed from literals by the main workflow.

Reply:
Changed.